Explorer/assignment-2/src/pages/Explorer/ExplorerPage.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from "react";
+import Header from "../../components/Header/Header";
+import Button from "../../components/Button/Button";
+import DestinationSection from "../../DestinationSection/DestinationSection";
+import cover from "../../assets/images/banner.webp";
+import "./ExplorerPage.css";
+import { useNavigate } from "react-router";
+import Footer from "../../components/Footer/Footer";
+import dropDown from "../../assets/images/images.png";
+import ExplorerService from "../../services/ExplorerService";
+import Select from "../../components/Select/Select";
+
+export default function ExplorerPage() {
+  const [placeData, setData] = useState([]);
+  const [option, setOptions] = useState("");
+  const navigate = useNavigate();
+  // To fetch all the data
+  useEffect(() => {
+    const fetchData = async () => {
+      const placeData = await ExplorerService.getAllPlaces();
+      setData(placeData);
+    };
+    fetchData();
+  }, []);
+
+  //  To navigate to another page on click
+  const handleExploreClick = () => {
+    if (option) {
+      navigate(`/details/${option}`);
+    }
+  };
+
+  const handleDropdown = (selectedValue) => {
+    setOptions(selectedValue);
+  };
+
+  return (
+    <>
+      <Header />
+      {/* Promo and search section */}
+      <div className="promo-section">
+        <div className="explorer-section">
+          <p className="explorer-heading">WELCOME TO EXPLORER</p>
+          <p className="explorer-description">
+            Your Adventure Travel Expert in the <span>SOUTH</span>
+          </p>
+          <div className="input-container">
+            <Select
+              id="selected-option"
+              options={placeData}
+              onChange={handleDropdown}
+            />
+
+            <img className="icon" src={dropDown} alt="Icon" />
+          </div>
+
+          <Button
+            className="explorer-button"
+            label={"EXPLORE"}
+            clicked={handleExploreClick}
+          ></Button>
+        </div>
+        <div className="explorer-image">
+          <img src={cover} alt="coverImage" />
```

Comment:
could you let me know if img could be a component

Reply:
Yes Durga.i have made use of it in 3rd exercise.I will update here